test/test_scenario_1.py

```diff
@@ -55,8 +62,11 @@ def test_tropo_delay(tmp_path):
         # get the results
         wet = gdal_open(wet_file)
         hydro = gdal_open(hydro_file)
-        true_wet = gdal_open(os.path.join(SCENARIO_DIR, "ERA5_wet_true.envi"))
-        true_hydro = gdal_open(os.path.join(SCENARIO_DIR, "ERA5_hydro_true.envi"))
+        true_wet = gdal_open(os.path.join(
+            SCENARIO_DIR, "wet.envi"), userNDV=0.)
+        true_hydro = gdal_open(os.path.join(
+            SCENARIO_DIR, "hydro.envi"), userNDV=0.)
+        print(true_wet)
```

Comment:
delete debug output
```suggestion
```

Reply:
@Askaholic fixed.

Reply:
Use the "Resolve conversation" button :)